client/src/components/navbar/index.js

```diff
@@ -1,12 +1,11 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```

Comment:
Do you need this after your changes ?

Reply:
Yes, I need the Router and the Link to Navigate using the navbar

Reply:
Events file is in a different pull request, resolved all comments.